Tasks/MSBuild/Helpers.ps1

```diff
@@ -60,6 +60,11 @@ function Invoke-BuildTools {
         [switch]$NoTimelineLogger
     )
 
+    if ($NuGetRestore)
+    {
+        Write-Warning (Get-LocalizedString -Key "The `"Restore NuGet Packages`" option is deprecated. To restore NuGet packages in your build, add a NuGet Installer task to your build definition.")
```

Comment:
A checkbox is unfortunately insufficient to work with VSTS-hosted NuGet feeds. We need to know the package source URIs ahead of time in order to configure authentication for them, and I don't think we want to grow *these* tasks into full-fledged NuGet restore tasks

Can you elaborate on "make sure the template corresponds with the change here."?

I don't think the _PS3 version existed when I started this work, so I didn't see it, sorry